nginx/metakgp.org

```diff
@@ -27,7 +27,7 @@ server {
         autoindex off;
 
 	# Make site accessible from everywhere
-	server_name _;
```

Comment:
Unfortunately I _think_ this causes things like locahost to stop working, which we want for development. Can you verify whether this is the case, and see if there's a way around it?

Reply:
Hey, I don't have docker installed on my computer. I did `wget localhost` on the server and that just gave me a 301 to the server URL. I will see if I have the space on my computer to set the whole thing up and check this.